source/efficient_mode/EfficientSymbiont.h

```diff
@@ -221,7 +221,7 @@ class EfficientSymbiont: public Symbiont {
    */
   void VerticalTransmission(emp::Ptr<Organism> host_baby) {
     if((my_world->WillTransmit()) && GetPoints() >= my_config->SYM_VERT_TRANS_RES()){ //if the world permits vertical tranmission and the sym has enough resources, transmit!
-      emp::Ptr<Organism> sym_baby = reproduce("vertical");
+      emp::Ptr<Organism> sym_baby = Reproduce("vertical");
```

Comment:
Do you mind adding a test for the EfficientSymbiont VerticalTransmission while you're at it?